sdk/keyvault/keyvault-admin/package.json

```diff
@@ -23,50 +23,40 @@
   "bugs": {
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
-  "main": "./dist/index.js",
-  "module": "dist-esm/src/index.js",
-  "types": "./types/keyvault-admin.d.ts",
+  "main": "./dist/commonjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/commonjs/index.d.ts",
   "engines": {
     "node": ">=18.0.0"
   },
   "files": [
-    "types/keyvault-admin.d.ts",
     "dist/",
-    "dist-esm/src",
     "README.md",
     "LICENSE"
   ],
-  "browser": {
-    "os": false,
-    "process": false
-  },
+  "browser": "./dist/browser/index.js",
```

Comment:
Do we need this field if we are not providing browser support? IIRC we removed and then re-added this field to support legacy Webpack versions that do not understand the `exports` field.